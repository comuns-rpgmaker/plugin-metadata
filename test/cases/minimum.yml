---
target: MZ
description: Minimal metadata
author: Me
help: This is a minimal metadata file.
