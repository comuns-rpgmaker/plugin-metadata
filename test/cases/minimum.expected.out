/*:
 * @target MZ
 * @author Me
 * @plugindesc Minimal metadata
 * @help This is a minimal metadata file.
 */
